.github/workflows/ci.yml

```diff
@@ -122,12 +122,14 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        version: [ latest, lts/*, lts/-1 ]
+        # check the minimum node version supported by the metrics script and the latest node version
+        # (assumes the versions between have backwards-compatible APIs)
+        version: [14.10.0, latest]
```

Comment:
Since `latest` might resolve to the odd number / temporary releases (like 21.x starting next fall), I still think it's a good idea to put `lts/*` in here.

Reply:
Yes, and that is deliberate since the APIs used here should be stable between the given range of versions so checking against the earliest and latest versions should cover that. If you're feeling like that's not reliable enough then I can add `lts/*` and `lts/-1` back in.